business_rules/operators.py

```diff
@@ -95,6 +95,10 @@ def matches_regex(self, regex):
     def non_empty(self):
         return bool(self.value)
 
+    @type_operator(FIELD_TEXT)
+    def contains_in(self, other_string):
```

Comment:
How about just "is_in"? Contains_in isnt't really a thing people say.

Reply:
Yup, it really fits what we are trying to do